spring-integration-core/src/main/java/org/springframework/integration/channel/DefaultHeaderChannelRegistry.java

```diff
@@ -259,6 +259,11 @@ public void run() {
 		}
 	}
 
+	@Override
+	public String getComponentType() {
+		return "channel";
```

Comment:
`header-channel-registry`.

I can argue that this class is not supposed to be an `IntegrationObjectSupport`.
But that's different story.